tests/compiletest.rs

```diff
@@ -3,7 +3,15 @@ extern crate compiletest_rs as compiletest;
 use std::path::{PathBuf, Path};
 use std::io::Write;
 
+macro_rules! eprintln {
+    ($($arg:tt)*) => {
+        let stderr = std::io::stderr();
+        writeln!(stderr.lock(), $($arg)*).unwrap();
+    }
```

Comment:
This is the first time I write a macro... will the curly braces here also be added at the expansion? IOW, will `stderr` be dropped right after the `writeln!` or will it stick around for the remainder of the scope in which the macro is used?